src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -3007,6 +2981,38 @@ long DirSize(DirectoryInfo d)
             }
         }
 
+        private static string GetPlatformMoniker()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                if (RuntimeInformation.ProcessArchitecture == Architecture.Arm64)
+                {
+                    return "win-arm64";
+                }
+                else
+                {
+                    return "win-x64";
+                }
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                return "osx-x64";
```

Comment:
M1?

Reply:
Not specifically, just moved some code around as I needed to reuse it, I don't know when it got in.
Feel free to buy one and try though ;)